bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html

```diff
@@ -0,0 +1,135 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_73" %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - Make Firefox your default browser') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_73') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% set android_url = firefox_adjust_url('android', campaign) %}
+{% set ios_url = firefox_adjust_url('ios', campaign) %}
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <aside id="modal" class="mzp-u-modal-content mzp-l-content">
+        <h2 class="modal-logo">Firefox</h2>
+        <h3>{{ _('Get Firefox on your Phone') }}</h3>
+
+        {% if show_send_to_device %}
+          {{ send_to_device(include_title=False, message_set='fx-mobile-download-desktop', class='vertical') }}
```

Comment:
I'm guessing this should be `fx-whatsnew`
```suggestion
          {{ send_to_device(include_title=False, message_set='fx-whatsnew', class='vertical') }}
```